openlibrary/plugins/openlibrary/tests/test_home.py

```diff
@@ -35,7 +35,7 @@ def test_about_template(self, render_template):
         posts = [web.storage({
             "title": "Blog-post-0",
             "link": "http://blog.openlibrary.org/2011/01/01/blog-post-0",
-            "pubdate": datetime.datetime(2011, 0o1, 0o1)
+            "pubdate": datetime.datetime(2011, 1, 1)
```

Comment:
While a little clearer, this is unrelated to Python 3

Reply:
$ __python3 -c "01"__  # --> SyntaxError: invalid token

Reply:
Are we going to enter dates in octal?

Reply:
https://github.com/internetarchive/openlibrary/pull/1466/files#diff-e9b7e7fd89beb0a8b81aad05951d6d38L37

Reply:
https://github.com/internetarchive/openlibrary/pull/1466/files#diff-e9b7e7fd89beb0a8b81aad05951d6d38L36

Reply:
The original code cannot have been deliberate octal, it made me chuckle, because it looks sensible as a zero-padded date, but what is the sequence for other dates going to be?: `(01, 02, 03, 04, 05, 06, 07, 8, 9, 10)`?  or perhaps  `..., 07, 010, ...` ? Either a copy-paste or visual formatting error/choice that just happened to work out.

I agree `datetime.datetime(2011, 1, 1)` is the correct fix here, but I am enjoying the idea of octal dates :)   February has 28 days, but sometimes 035 when over-compensating... etc

Reply:
These changes were already merged in #1492